optaplanner-quarkus-integration/optaplanner-quarkus/pom.xml

```diff
@@ -17,6 +17,7 @@
     <module>runtime</module>
     <module>deployment</module>
     <module>integration-test</module>
+    <module>devui-integration-test</module>
     <module>drl-integration-test</module>
```

Comment:
Any chance we can merge these? This number of "not really" modules is growing, and each is only very small - yet each brings Maven overhead.

Reply:
Nope; I tried to (first in integration-tests, second in deployment). In integration-tests, it failed the build because you cannot mix extension test runners with unit test runners. In deployment, it failed the build because vert.x/web is not a required dependency (I think).